config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
why ? I tried with the same kernel path, and it worked for me. What did I miss ?